docs/cinder-csi-plugin/examples.md

```diff
@@ -256,6 +257,61 @@ $ openstack volume list
 | bf8f9ae9-87b4-42bb-b74c-ba4645634be6 | pvc-4699fa78-4149-4772-b900-9536891fe200 | available |    1 |                                                 |
 ```
 
+## Use Topology
+
+Check [sample app](../../examples/cinder-csi-plugin/topology/example.yaml) for how to use Topology.
+
+Prerequisites:
+Deployed CSI and with feature gate Topology enabled ("--feature-gates=Topology=true", which is by default True now)
```

Comment:
its better to add CSI provisioner for better clarity here

Reply:
done